contracts/PoaOperator.sol

```diff
@@ -123,6 +132,7 @@ contract PoaOperator is Adminable {
     if (_signerAddr == address(0) && _tenderAddr == 0) {
       slot.activationEpoch = uint32(lastCompleteEpoch + 3);
       slot.eventCounter++;
+      takenSlots = setTakenSlotBitmap(takenSlots, _slotId, false);
```

Comment:
Can we have some unit tests checking taken slots bitmap is set correctly after logout?

E.g. this test fails for me (I've temporary made `setTakenSlotBitmap` public):

```js
const bits110 = '0x0000000000000000000000000000000000000000000000000000000000000006';
assert(
  await operator.setTakenSlotBitmap(bits110, 253, false)
).equal(BN(2));
```

while 

```js
assert(
  await operator.setTakenSlotBitmap(bits110, 254, false)
).equal(BN(4));
```

works

Reply:
checking ...

Reply:
@troggy great catch 🎉 logic was broken, deleting all bits on the right, as shifting filled up the bitmask with 0s.